rlp/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rlp"
-version = "0.5.0"
+version = "0.6.0"
```

Comment:
wait, it doesn't need a major bump?

Reply:
I thought, because it depends on `primitive-types`, and that has been bumped as part of this

Reply:
that's a dev-dep

Reply:
:facepalm:

Reply:
See https://github.com/paritytech/parity-common/pull/552#discussion_r661848985 for a possible mitigation